docs/administer/rbac_pod_scheduling.md

```diff
@@ -0,0 +1,42 @@
+# Pod scheduling policies with RBAC
```

Comment:
RBAC page https://docs.percona.com/everest/administer/rbac.html#built-in-role

Build-in Admin role has to be extended with the following string:
```
p, role:admin, pod-scheduling-policies, *, *
```

Reply:
In addition it is worth to mention that PSP doesn't belong to or limited by any namespace. In general each PSP can be used in all namespaces.

Reply:
Fixed.